js/main.js

```diff
@@ -0,0 +1,119 @@
+(function() {
+
+    function loadPics() {
+        //supplied values from flickr
+        var api_key = '74e47a159e15cbcb6139ba9c9df64c13';
+        var secret = 'c73c5dcbac8e4a96';
+        var method = "flickr.photos.search";
+        var user_id = "138698049@N03";
+        var format = '&format=json';
+        var url = "https://api.flickr.com/services/rest"+"?method="+method+"&api_key="+api_key+"&user_id="+user_id+format+'&nojsoncallback=1';
+
+        var xmlhttp = new XMLHttpRequest();
+
+        xmlhttp.onreadystatechange = function () {
+            var farm, server, id, title, secret, picUrl, list, img, items;
+
+            if (xmlhttp.readyState == 4 && xmlhttp.status == 200) {
+                //make an empty array
+                var pics = [];
+
+                //parse request to json
+                var myResp = JSON.parse(xmlhttp.response);
+                //console.log(myResp);
```

Comment:
Remove all comments. Code should be understandable to the reader. If comments are necessary to describe a block, consider refactoring the code block.